main/export-options.js

```diff
@@ -60,6 +62,7 @@ const getExportOptions = () => {
         }
       }
     } catch (error) {
+      dialog.showErrorBox(`Something went wrong while loading ${json.pluginName}`, error.stack);
```

Comment:
```suggestion
      dialog.showErrorBox(`Something went wrong while loading “${json.pluginName}”`, error.stack || error || 'Unknown error');
```

It could fail with a native C error if the `.stack` is not defined, which could happen. I experienced that with Caprine.

Reply:
Nice little utility. I abstracted the method and used it 👍